newIDE/app/src/Export/GenericExporters/OnlineWebExport/OnlineGameLink.js

```diff
@@ -201,34 +278,15 @@ const OnlineGameLink = ({
           }
         );
         setGame(updatedGame);
-        const { userSlug, gameSlug } = partialGameChange;
-        if (userSlug && gameSlug && userSlug === profile.username) {
-          try {
-            await setGameSlug(
-              getAuthorizationHeader,
-              id,
-              game.id,
-              userSlug,
-              gameSlug
-            );
-            setSlug({ username: userSlug, gameSlug: gameSlug, createdAt: 0 });
-          } catch (error) {
-            console.error(
-              'Unable to update the game slug:',
-              error.response || error.message
-            );
-            showErrorBox({
-              message:
-                i18n._(
-                  t`Unable to update the game slug. A slug must be 6 to 30 characters long and only contains letters, digits or dashes.`
-                ) +
-                ' ' +
-                i18n._(t`Verify your internet connection or try again later.`),
-              rawError: error,
-              errorId: 'game-slug-update-error',
-            });
-            return false;
-          }
+        // Then set the authors.
+        const authorSuccess = await tryUpdateAuthors(i18n);
+        if (!authorSuccess) {
+          return false;
+        }
+        // Then set the slug if it has been filled.
+        const slugSuccess = await tryUpdateSlug(partialGameChange, i18n);
```

Comment:
When the author update doesn't succeed, the game is updated but not the slug. Is it ok?

Reply:
Good catch, we should probably run both in parallel :)

Reply:
yep! will do